docs/src/main/paradox/docs/delta/api/multi-fetch.md

```diff
@@ -0,0 +1,58 @@
+# Multi fetch
+
+The multi-fetch operation allows to get in a given format multiple resources that can live in multiple projects.
+
+The response includes a resources array that contains the resources in the order specified in the request. 
+The structure of the returned resources is similar to that returned by the fetch API. 
+If there is a failure getting a particular resource, the error is included in place of the resource.
+
+This operation can be used to return every type of resource.
+
+@@@ note { .tip title="Authorization notes" }
+
+When performing a request, the caller must have `resources/read` permission on the project each resource belongs to.
+
+Please visit @ref:[Authentication & authorization](authentication.md) section to learn more about it.
+
+@@@
+
+## Payload
+
+```
+GET /v1/multi-fetch/resources
+
+{
+  "format": {format}
+  "resources": [
+    {
+      "id": "{id}",
+      "project": "{project}"
+    },
+    ...
+  ]
+}
+```
+
+where...
+
+- `{format}`: String - the format we expect for the resources in the response. 
+Accepts the following values: source (to get the original payload), compacted, expanded, n-triples, dot
+- `{project}`: String - the project (in the format 'myorg/myproject') where the specified resource belongs. This field
+  is optional. It defaults to the current project.
+- `{id}`: Iri - the @id value of the resource to be returned. Can contain a tag or a revision.
+
+## Example
+
+The following example shows how to perform a multi-fetch and an example of response
+containing errors (missing permissions and resource not found).
+As a response, a json is returned.
```

Comment:
It may be interesting to put emphasis on the fact that this is unlike other endpoints which return json-ld